packages/gql/src/pool-snapshots/snapshots.ts

```diff
@@ -0,0 +1,12 @@
+import { gql } from "graphql-tag";
+
+export const PoolSnapshots = gql`
+  query PoolSnapshot($pool: String, $timestamp: Int) {
+    poolSnapshots(where: { pool: $pool, timestamp_gte: $timestamp }) {
+      pool {
+        id
+      }
+      amounts
```

Comment:
why not get the total liquidity here? The instantaneous subgraph value might be off but the point of the snapshot is that that value is usually gonna be correct. See [this quick query I ran](https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-v2/graphql?query=query+%7B%0A++poolSnapshots(where:+%7Bpool:%220x87a867f5d240a782d43d90b6b06dea470f3f8f22000200000000000000000516%22,+timestamp_gt:+1692884610%7D,+first:1)+%7B%0A++++liquidity%0A++++pool+%7B%0A++++++id%0A++++%7D%0A++%7D%0A%7D): 

```graphql
query {
  poolSnapshots(where: {pool:"0x87a867f5d240a782d43d90b6b06dea470f3f8f22000200000000000000000516", timestamp_gt: 1692884610}, first:1) {
    liquidity
    pool {
      id
    }
  }
}
```
result:
```json
{
  "data": {
    "poolSnapshots": [
      {
        "liquidity": "17571798.52955511399359994811209584",
        "pool": {
          "id": "0x87a867f5d240a782d43d90b6b06dea470f3f8f22000200000000000000000516"
        }
      }
    ]
  }
}
```

Reply:
Because it yesterday it was returning the same wrong value as the query we were running before

Reply:
you mean in the pools subgraph, right? yesterday there was no snapshot for yesterday yet 😅